### DISCUSSION TITLE
New and just setup, does it show data for car history. (newbie)

### DISCUSSION THREAD

**Discussion by OzBrown** [teslamate-org/teslamate#1780] [Getting started]
I just setup this application and found it looks great, have been reading the Discussions and answers, lots to re read again as I understand this application more.

I got my Tesla 25 June, was looking at the data and could only see from now, 1 July, does it pull data from the first day or is it possible to see it, as the car has only done 653klm's 

Will start to look at the code as I am retired, used to program etc so will keep me busy :) and hope to add etc as I can

Regards
Lindsay

**Comment by cwanja** [comment]
When did you setup TeslaMate to start polling the data? From that point forward is the only data that it can pull. It cannot ping back into Tesla's data to pull the statistics from any point before TeslaMate started polling the car.
